### PR TITLE
Fixed Typos

### PR DESCRIPTION
**Pull request by AyushBherwani1998**

### CODE REVIEW

**Review by pawanpraka1 — APPROVED** [review]
looks good. Can you pull the latest changes and update the pull request.

**Comment by pawanpraka1** [comment]
also can you DCO signoff the commit.
```
Sign your work
We use the Developer Certificate of Origin (DCO) as an additional safeguard for the OpenEBS project. This is a well established and widely used mechanism to assure that contributors have confirmed their right to license their contribution under the project's license. Please read dcofile. If you can certify it, then just add a line to every git commit message:

  Signed-off-by: Random J Developer <random@developer.example.org>
Use your real name (sorry, no pseudonyms or anonymous contributions). The email id should match the email id provided in your GitHub profile. If you set your user.name and user.email in git config, you can sign your commit automatically with git commit -s.

You can also use git aliases like git config --global alias.ci 'commit -s'. Now you can commit with git ci and the commit will be signed.
```

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# libcstor
+# Libcstor
```
Comment (vishnuitta) [inline README.md]: this is better to small case itself

Comment (AyushBherwani1998) [reply README.md]: I have made the appropriate changes

**Comment by vishnuitta** [comment]
Needs DCO signoff

**Comment by AyushBherwani1998** [comment]
> 
> 
> Needs DCO signoff

I have signed the DCO

**Review by kmova — APPROVED** [review]

**Comment by NisantaSahoo** [comment]
Thanks for taking part in Hacktoberfest 2019. Please fill up this form to claim your swag https://hubs.ly/H0l5ldm0

**Pull request merged**